src/plugin.js

```diff
@@ -53,9 +53,9 @@ export function frameworkImportPath(framework, options) {
 		case "react":
 		case "vue":
 		case "vue-next":
-		case "solid":
 		case "preact":
 			return `/dist/${options.importMode}/icons/`;
+		case "solid":
```

Comment:
You can even remove this one, it's just gonna fallback into `default`

Reply:
True, i feel like it's more clear that solid is "tested" by including it here though.

Reply:
In that case I’d have to have to add Svelte in there too
I feel like it’s easier to maintain a blacklist than both a blacklist and a whitelist though, more flexible if new frameworks come to be supported by Lucide

Reply:
makes sense.